jobs/update-project-catalogue/values.yaml

```diff
@@ -0,0 +1,9 @@
+# Default values for update-project-catalogue.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: quay.io/ebi-ait/update-project-catalogue
+  tag: latest
+  pullPolicy: Always
+schedule: 0 0 * * *
```

Comment:
could be run every 10pm, `0 22 * * *` so that our backup will contain the latest updates.